src/Components/CreateImageWizardV2/steps/TargetEnvironment/Azure/index.tsx

```diff
@@ -0,0 +1,201 @@
+const Azure = () => {
+  const dispatch = useAppDispatch();
+  const shareMethod = useAppSelector((state) => selectAzureShareMethod(state));
+  const tenantId = useAppSelector((state) => selectAzureTenantId(state));
+  const subscriptionId = useAppSelector((state) =>
+    selectAzureSubscriptionId(state)
+  );
+  const resourceGroup = useAppSelector((state) =>
+    selectAzureResourceGroup(state)
+  );
+
+  return (
+    <Form>
+      <Title headingLevel="h2">Target environment - Microsoft Azure</Title>
+      <Text>
+        Upon build, Image Builder sends the image to the selected authorized
+        Azure account. The image will be uploaded to the resource group in the
+        subscription you specify.
+      </Text>
+      <Text>
+        To authorize Image Builder to push images to Microsoft Azure, the
+        account owner must configure Image Builder as an authorized application
+        for a specific tenant ID and give it the role of &quot;Contributor&quot;
+        for the resource group you want to upload to. This applies even when
+        defining target by Source selection.
+        <br />
+        <Button
+          component="a"
+          target="_blank"
+          variant="link"
+          icon={<ExternalLinkAltIcon />}
+          iconPosition="right"
+          isInline
+          href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow"
+        >
+          Learn more about OAuth 2.0
+        </Button>
+      </Text>
+      <FormGroup label="Share method:">
+        <Radio
+          id="radio-with-description"
+          label="Use an account configured from Sources."
+          name="radio-7"
+          description="Use a configured sources to launch environments directly from the console."
+          isChecked={shareMethod === 'sources'}
+          onChange={() => {
+            dispatch(changeAzureSource(undefined));
+            dispatch(changeAzureTenantId(''));
+            dispatch(changeAzureShareMethod('sources'));
+          }}
+        />
+        <Radio
+          id="radio"
+          label="Manually enter the account information."
+          name="radio-8"
+          isChecked={shareMethod === 'manual'}
+          onChange={() => {
+            dispatch(changeAzureSource(undefined));
+            dispatch(changeAzureTenantId(''));
+            dispatch(changeAzureSubscriptionId(''));
+            dispatch(changeAzureShareMethod('manual'));
+          }}
+        />
+      </FormGroup>
+      {shareMethod === 'sources' && (
+        <>
+          <AzureSourcesSelect />
+          <SourcesButton />
+          <Gallery hasGutter>
+            <GalleryItem>
+              <FormGroup label="Azure Tenant GUID">
+                <TextInput
+                  aria-label="Azure Tenant GUID"
+                  readOnlyVariant="default"
+                  isRequired
+                  id="tenant id"
+                  value={tenantId}
+                />
+              </FormGroup>
```

Comment:
Azure Tenant GUID wasn't marked as required next to its label. We'll just have to move the `isRequired` from the TextInput to the FormGroup

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/48c20d8c-963b-4ef5-9579-2b2c6da5cda0)


```suggestion
              <FormGroup label="Azure Tenant GUID" isRequired>
                <TextInput
                  aria-label="Azure Tenant GUID"
                  readOnlyVariant="default"
                  id="tenant id"
                  value={tenantId}
                />
              </FormGroup>
```